contracts/ValidatorSetAuRa.sol

```diff
@@ -1,16 +1,15 @@
 pragma solidity 0.5.2;
 
 import "./abstracts/ValidatorSetBase.sol";
+import "./RandomAuRa.sol";
 
 
 contract ValidatorSetAuRa is ValidatorSetBase {
 
-    // TODO: add a description for each function
-
+    // TODO: add a description for each function)
     // ============================================== Constants =======================================================
-
-    uint256 public constant STAKING_EPOCH_DURATION = 120960; // 1 week in blocks (5 seconds per block)
-    uint256 public constant STAKE_WITHDRAW_DISALLOW_PERIOD = 4320; // 6 hours in blocks (staking epoch last blocks)
+    bytes32 constant private STAKE_WITHDRAW_DISALLOW_PERIOD = "stackWithdrawlDisallowPeriod";
+    bytes32 constant private STAKING_EPOCH_DURATION = "stakingEpochDuration";
```

Comment:
These constants should be in the `Private` section of the file. The `Constants` section is for public constants. Also, they should be calculated as keccak256. E.g.: https://github.com/poanetwork/pos-contracts/blob/240e6e5e2e695a7761267185bc52adf950fa2951/contracts/RandomAuRa.sol#L201